src/app/report-dashboard/report-dashboard.component.html

```diff
@@ -21,23 +21,24 @@ <h1 class="w-min whitespace-nowrap text-3xl font-bold">Rapportage dashboard</h1>
     <div echarts [options]="mostUsagesPieOptions" class="w-3/6 h-[400px] border-2 border-secondary-purple-light p-4 rounded-lg shadow-lg"></div>
```

Comment:
Most usages laat dezelfde ruimte zien, maar in meerdere pies

Reply:
Heb je het met de location seeder in mijn backend branch (feat/reports) getest, want ik heb die aangepast omdat hij eerst meerdere locations met dezelfde naam in de database zette.